README.md

```diff
@@ -31,14 +31,15 @@ First you will need a `docker-compose.yaml` file, such as the example included.
 * Copy [sponsorblockcast.service](/sponsorblockcast.service) to `/usr/lib/systemd/system/sponsorblockcast.service`.
 
 ## Usage
-Run `sponsorblockcast` from a terminal or activate the service with `systemd enable --now sponsorblockcast`
+Run `sponsorblockcast` from a terminal or activate the service with `systemd enable sponsorblockcast && systemctl start sponsorblockcast`
```

Comment:
Any particular reason for this change besides being more explicit?

Reply:
Also, noticing now I'd written the command as `systemd` instead of `systemctl`, oops!

Reply:
The `--now` option wouldn't work for me on Raspberry Pi OS. 

I'm newly learning to use `git` and realised too late that these changes should have been in separate commits.